### PR TITLE
ASC-1655 Add Ceph Scenario to Staging Job

### PR DESCRIPTION
**Pull request by ryan-rs**
Add the "ceph" scenario to the system staging job so that we can start
creating Molecule tests for Ceph backends. Also, I dropped a redundant
image specification.

Issue: [ASC-1655](https://rpc-openstack.atlassian.net/browse/ASC-1655)

### CODE REVIEW

File: rpc_jobs/rpc_openstack.yml [hunk 1]
```diff
@@ -604,10 +604,9 @@
     image:
       - bionic_mnaio_no_artifacts:
           SLAVE_TYPE: "nodepool-ubuntu-bionic-om-io2"
-      - xenial_mnaio_no_artifacts:
```
Comment (johnduarte) [inline rpc_jobs/rpc_openstack.yml]: These two changes are completely unrelated. Can you break them into separate commits?

Comment (ryan-rs) [reply rpc_jobs/rpc_openstack.yml]: @johnduarte Updated.
